toolkit/docs/formats/imageconfig.md

```diff
@@ -174,7 +174,7 @@ A sample `ParitionSettings` entry using `overlay` algorithm:
 `OverlayBaseImage` represents the base image when `overlay` algorithm is used.
 
 ### EnableGrubMkconfig
-EnableGrubMkconfig is a optional boolean that controls whether the image uses grub2-mkconfig to generate the boot configuration (/boot/grub2/grub.cfg) or not. If EnableGrubMkconfig is specified, only valid values are `true` and `false`. Default is `false`. 
+EnableGrubMkconfig is a optional boolean that controls whether the image uses grub2-mkconfig to generate the boot configuration (/boot/grub2/grub.cfg) or not. If EnableGrubMkconfig is specified, only valid values are `true` and `false`. Default is `false`.
```

Comment:
I don't understand why this bit shows up as a cherry-picked change when it was already merged three days ago into 3.0-dev: https://github.com/microsoft/CBL-Mariner/commit/685a93c2bb0f1ab8dd124ccdf050704ac1c086d6#diff-4276d3506413499e619bc8648536a76b428854629b153e9f5fcc3dae95d8f7feR177

Reply:
.... I missed the space after the period getting trimmed. It is not being introduced here.. disregard.